### PR TITLE
Dependency fixes to pass the travis.ci build - Fixes #73

### PR DESCRIPTION
**Pull request by jfucci**

### CODE REVIEW

**Comment by jfucci** [comment]
Fixes #73

File: server/api/project/project.controller.js [hunk 1]
```diff
@@ -76,3 +79,36 @@ exports.destroy = function(req, res) {
...
+    var file = files.file[0];
+    var name = file.path.substring(file.path.lastIndexOf('/')).substring(1);
+    var cwd = process.cwd();
+    var path = cwd + '/server/static/uploads/' + req.params.username + '/' + req.params.project;
+    var destPath = path + '/' + name;
+    if(!fs.existsSync(path)){
+      mkdirp.sync(path);
+    }
+    fs.rename(file.path, destPath, function (err) {
+      if (err) console.error(err)
```
Comment (agundy) [inline server/api/project/project.controller.js]: These should return errors to the clients. Console errors on the server are just going to be eaten in the logs and never seen

File: server/routes.js [hunk 2]
```diff
@@ -15,13 +16,15 @@ module.exports = function(app) {
   app.use('/api/things', require('./api/thing'));
   app.use('/api/users', require('./api/user'));
 
+  app.use('/uploads', express.static( __dirname + "/static/uploads" ));
```
Comment (agundy) [inline server/routes.js]: Could we move this outside of the project directory? the current deployment system relies on the ability to move the entire project structure but photos shouldn't be reset between versions.

File: bower.json [hunk 3]
```diff
@@ -15,10 +15,15 @@
     "lodash": "~2.4.1",
     "angular-ui-router": "~0.2.10",
     "angular-bootstrap": "~0.11.0",
-    "codemirror": "~5.1.0"
+    "codemirror": "~5.1.0",
+    "angular-file-upload": "~1.1.5",
+    "ng-file-upload": "~3.3.1",
+    "ng-file-upload-shim": "~3.3.1"
```
Comment (agundy) [inline bower.json]: Are we actually using all three of these? Couldn't tell from the code but just want verification

**Pull request merged**